standard.gemspec

```diff
@@ -20,5 +20,5 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "rubocop", "1.22.3"
-  spec.add_dependency "rubocop-performance", "1.11.5"
+  spec.add_dependency "rubocop-performance", "~> 1.12"
 end
```

Comment:
We have made an intentional decision to lock to specific releases of rubocop and rubocop-performance, in order that standard's behavior at any given version is reproducible in all cases for a given target Ruby. If you'd like to bump to the latest rubocop-performance release, that's great (we just hadn't gotten to it yet), but please lock to the specific version.